.gitignore

```diff
@@ -1,3 +1,4 @@
+# The code completion language server cache files.
 .ccls-cache/
-src/*.o
-imprimis_gameserver
```

Comment:
It seems that the binary output should still be gitignored (in the case that a developer has not run the install command).

Reply:
I will add the lines back to the .gitignore file. As for building it in Windows, it probably will not work since I did not get the chance to test it in Windows.